.github/workflows.src/build.targets.yml

```diff
@@ -11,14 +11,12 @@ targets:
           platform_version: buster
           family: debian
           runs_on: [package-builder, self-hosted, linux, x64]
-          docker_arch: linux/amd64
         - name: debian-buster-aarch64
           arch: aarch64
           platform: debian
           platform_version: buster
           family: debian
           runs_on: [package-builder, self-hosted, linux, arm64]
-          docker_arch: linux/arm64
```

Comment:
Why?

Reply:
https://github.com/edgedb/edgedb-docker/pull/91 changed to use `bookworm` for Docker base, and updating `docker_arch` to `debian-bookworm-*` ensures the correct CI job dependency and version referring.